Pear/Onboarding/AnswerPromptViewController.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  AnswerPromptViewController.swift
+//  Pear
+//
+//  Created by Amy Chin Siu Huang on 3/28/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class AnswerPromptViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private let titleLabel = UILabel()
+    private let questionLabel = UILabel()
+    private let responseTextView = UITextView()
+    private let characterCountLabel = UILabel()
+    private let saveButton = UIButton()
+
+    // MARK: - Private Data Vars
+    private var prompt: Prompt?
+    private var index: Int?
+    private weak var delegate: AddProfilePromptDelegate?
+
+    init(prompt: Prompt, delegate: AddProfilePromptDelegate, index: Int) {
+        super.init(nibName: nil, bundle: nil)
+        self.prompt = prompt
+        self.delegate = delegate
+        self.index = index
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .backgroundLightGreen
+        navigationController?.navigationBar.isHidden = true
+
+        backButton.setImage(UIImage(named: "backArrow"), for: .normal)
+        backButton.addTarget(self, action: #selector(backButtonPressed), for: .touchUpInside)
+        view.addSubview(backButton)
+
+        titleLabel.text = "Enter your response"
+        titleLabel.font = ._24CircularStdMedium
+        titleLabel.textAlignment = .center
+        view.addSubview(titleLabel)
+
+        guard let selectedPrompt = prompt?.promptQuestion else { return }
```

Comment:
Best to delete this after you make `prompt` a non-optional. Otherwise, it seems weird to me that we `return` inside a `viewDidLoad`.